source/core/src/main/com/csse3200/game/entities/factories/NPCFactory.java

```diff
@@ -141,18 +140,17 @@ public static Entity createBaseNPC(Entity target) {
     AITaskComponent aiComponent =
         new AITaskComponent()
             .addTask(new WanderTask(new Vector2(2f, 2f), 2f))
-            .addTask(new MobAttackTask(2, 40))
-                .addTask(new MobDeathTask(2));
+            .addTask(new MobAttackTask(2, 40));
     Entity npc =
         new Entity()
             .addComponent(new PhysicsComponent())
             .addComponent(new PhysicsMovementComponent())
             .addComponent(new ColliderComponent())
-            .addComponent(new HitboxComponent().setLayer(PhysicsLayer.NPC))
-            .addComponent(new TouchAttackComponent(PhysicsLayer.HUMANS, 1.5f))
+            .addComponent(new HitboxComponent().setLayer(PhysicsLayer.XENO))
```

Comment:
Probably should announce the existence of a new physical layer so that everyone is aware of it and can modify their code if necessary